Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM dlanguage/dmd:2.080.0
-
-RUN apt-get update && \
-    apt-get install -y jq && \
-    apt-get purge --auto-remove -y && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
-
+FROM ubuntu:24.04
+RuN apt update
```

Comment:
You could combine all the apt commands in a single layer using && and \
This reduces the image size